app/views/how-tos/filters.html

```diff
@@ -0,0 +1,108 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+Filters - NHS prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-three-quarters">
+    <h1>Filters</h1>
+    <p class="nhsuk-lede-text">
+      Change how answers are shown.
+      </p>
+
+    {{ insetText({ html: '
+    <p>
+      To use filters, you need to know how to
+      <a href="passing-data-page">pass data from page to page</a>.
+    </p>
+    ' })}}
+
+    <p>
+      The kit stores data from all answers that users give in a prototype, so
+      that you can use or show the answers later. To change the format of how
+      these answers appear, you can apply filters. You can:</p>
+
+    <ul class="nshuk-list nhsuk-list-bullet">
+      <li>use existing Nunjucks filters</li>
+      <li>create a Nunjucks filter</li>
+      <li>use filters from other projects (recommended for advanced users only)</li>
+    </ul>
+    <h2 class="nhsuk-heading-m">Use existing Nunjucks filters</h2>
+    <p>
+      You can
+      <a
+        href="https://mozilla.github.io/nunjucks/templating.html#builtin-filters"
+        >use existing Nunjucks filters</a
+      >
+      in the kit. For example, you can use the Nunjucks
+      <code>upper</code> filter to change the format of text to upper case:
+    </p>
+
+    <pre
+      class="app-pre"
+    ><code class="app-code">{{ "{{data['name'] | upper }}" | escape }}</code></pre>
+
+    <h2 class="nhsuk-heading-m">Create a Nunjucks filter</h2>
+    <p>
+      Add your own filters to the <code>filters.js</code> file. Filters are
+      written in JavaScript.
+    </p>
+    <pre
+      class="app-pre"
+    ><code class="app-code">module.exports = function (env) { /* eslint-disable-line func-names,no-unused-vars */
+  const filters = {};
+
+  filters.uppercase = function (content) {
+    return content.toUpperCase()
+  }
+  return filters;
+};</code>
```

Comment:
This duplicates one of the built-in filters - maybe we can find another example which would be useful but isn't built in? Maybe a filter which purely makes the first letter uppercase? `test test` -> `Test test`?

Reply:
I think that's also a built in filter - [Capitalize](https://mozilla.github.io/nunjucks/templating.html#capitalize) - though I usually make my own that doesn't affect other capitalisation.

Reply:
Agree with all of this, I think that the DIY version should be the 'change first letter' as that is something legit useful in check your answer pages.

Reply:
>I think that's also a built in filter - [Capitalize](https://mozilla.github.io/nunjucks/templating.html#capitalize) - though I usually make my own that doesn't affect other capitalisation.

That one confusingly capitalises every word (including "and") so is next to useless... 🤦

Reply:
I think you're of the [title filter](https://mozilla.github.io/nunjucks/templating.html#title) - which is arguably better named.

As an aside, here's my sentenceCase filter which just modifies the first char:

```js
/**
 * Convert string to sentence case
 * @param {string} input - String to convert
 * @returns {string} Sentence case string
 */
const sentenceCase = (input) => {
  if (!input) return '';
  if (typeof input !== 'string') return input;
  return input.charAt(0).toUpperCase() + input.slice(1);
};
```

Reply:
Ah you're right, `capitalize` is annoying for a different reason - it capitalises the first letter but downcases everything else.

So `an iPad` gets transformed to `An ipad`.

Reply:
have added this to the latest push and also tweaked the function so that it aligns with the other one.

Reply:
Have added this example into the new version - closing this comment and will ask for new review